src/antsibull_docs/lint_plugin_docs.py

```diff
@@ -74,8 +74,8 @@ def __exit__(self, type_, value, traceback_):
 class CollectionFinder:
     def __init__(self):
         self.collections = {}
-        stdout = sh.Command('ansible-galaxy')('collection', 'list', '--format', 'json').stdout
-        raw_output = stdout.decode('utf-8', errors='surrogateescape')
+        p = log_run(['ansible-galaxy', 'collection', 'list', '--format', 'json'])
+        raw_output = p.stdout.decode('utf-8', errors='surrogateescape')
         data = json.loads(_filter_non_json_lines(raw_output)[0])
```

Comment:
`p.stdout` is already a string.

```suggestion
        p = log_run(['ansible-galaxy', 'collection', 'list', '--format', 'json'])
        data = json.loads(_filter_non_json_lines(p.stdout)[0])
```

Reply:
Right...

Reply:
8b35cf4c6d40caec7c62806369cb6a65407538ba